app/services/db/piv_cac_configuration.rb

```diff
@@ -0,0 +1,18 @@
+module Db
+  class PivCacConfiguration
+    def self.create(user, x509_dn_uuid, name = x509_dn_uuid, issuer = nil)
+      user.piv_cac_configurations.create!(x509_dn_uuid: x509_dn_uuid,
+                                          name: name,
+                                          x509_issuer: issuer)
+    end
+
+    def self.delete(user_id, cfg_id)
+      ::PivCacConfiguration.where(user_id: user_id, id: cfg_id).delete_all
+    end
+
+    def self.find_user_by_x509(x509_dn_uuid)
+      piv_cac_config = ::PivCacConfiguration.find_by(x509_dn_uuid: x509_dn_uuid)
+      piv_cac_config&.user
```

Comment:
I know this was picked up and moved, but do we need the intermediate variable here?

```suggestion
      ::PivCacConfiguration.find_by(x509_dn_uuid: x509_dn_uuid)&.user
```